ui/app/__mocks__/react-router-dom.js

```diff
@@ -3,4 +3,9 @@ const originalModule = jest.requireActual('react-router-dom');
 module.exports = {
   ...originalModule,
   useHistory: jest.fn(),
+  useLocation: jest.fn(() => {
```

Comment:
presumably when we expand test coverage in our other areas of the repo we'll need to mock this out in those files to override the pathname?

Reply:
That's right. My assumption is that most of the tests will be fine with this path, but if they need to use a different one, they can just do this:
```
jest.mock('react-router-dom', () => {
  const originalModule = jest.requireActual('react-router-dom');
  return {
    ...originalModule,
    useHistory: jest.fn(),
    useLocation: jest.fn(() => {
      return {
        pathname: '/feature-z/page-1',
      };
    }),
  }
});
```